sw/device/silicon_owner/tock/kernel/src/main.rs

```diff
@@ -42,9 +42,12 @@ use rv32i::csr;
 
 pub mod io;
 mod otbn;
+/*
+ * This could be reinstated if test code is brought in, see
+ * https://github.com/lowRISC/opentitan/issues/22737
 #[cfg(test)]
 mod tests;
-
+*/
```

Comment:
We will not put tests into this crate for the forseeable future, so delete this entirely rather than commenting it out.

Unit tests for drivers/components within the kernel belong in the driver/component crates, and integration tests that run on hardware belong in separate test crates under `sw/device/silicon_owner/tock/tests`.

Reply:
sure, done.